recipes/r-profilemodel/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# R refuses to build packages that mark themselves as Priority: Recommended
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
```

Comment:
Same as in other PR. Could you please remove above three lines? They are not needed.